### PR TITLE
Fix mysql duplicate key bug 

### PR DESCRIPTION
**Pull request by jimthrive**
See issue https://github.com/zdennis/activerecord-import/issues/548.

### CODE REVIEW

**Comment by jimthrive** [comment]
Looks like a travis issue:

```
ERROR:  While executing gem ... (Gem::RemoteFetcher::UnknownHostError)
    timed out (https://api.rubygems.org/specs.4.8.gz)
```

**Comment by oniofchaos** [comment]
Ideally there'd be some sort of regression test that prevents the described bug from happening again in the future

File: lib/activerecord-import/adapters/mysql_adapter.rb [hunk 1]
```diff
@@ -71,14 +71,12 @@ def pre_sql_statements( options)
 
   # Add a column to be updated on duplicate key update
   def add_column_for_on_duplicate_key_update( column, options = {} ) # :nodoc:
-    if options.include?(:on_duplicate_key_update)
+    if options.include?(:on_duplicate_key_update) && !options[:ignore] && !options[:on_duplicate_key_ignore]
```
Comment (jkowens) [inline lib/activerecord-import/adapters/mysql_adapter.rb]: With this change, I think we could simplify this to:

`if options[:on_duplicate_key_update] && !options[:ignore] && !options[:on_duplicate_key_ignore]`

**Comment by jkowens** [comment]
This has been released in v1.0.0 🎉

**Pull request merged**